src/balance.ts

```diff
@@ -0,0 +1,35 @@
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { BN } from "@polkadot/util";
+import type { Probot } from "probot";
+
+import { getChainConfig } from "./chain-config";
+import { balanceGauge } from "./metrics";
+import { TipNetwork } from "./types";
+
+export const updateAllBalances = async (tipBotAddress: string, log: Probot["log"]): Promise<void> => {
+  const networks: TipNetwork[] = ["localpolkadot", "localkusama", "kusama", "polkadot"];
```

Comment:
afaiu this will be running on prod? 
therefore 2 dumb questions
1. do we need on prod to update local* networks?
2. is the tipBotAddress same for each network?

i guess on prod we will see local* networks failed to check because of that

i wonder why we need to update All if we run one instance per its address/network and thus they will run updateBalance for itself?

Reply:
> do we need on prod to update local* networks?
> i guess on prod we will see local* networks failed to check balance

We don't need `local*`, my idea was to not do anything special for those and just let it fail once to check balance.
But maybe I better change it so we don't have errors every time the app is restarted.

---

> is the tipBotAddress same for each network?
> The answer to (2) should be yes.

I believe it is a yes.

---

> if we run one instance per its address/network

My understanding is that we run one instance per all networks (so Polkadot and Kusama in our case), using a single instance and a single seed for accounts on both those networks. So I wanted to update all of those networks on startup.

Reply:
ah right, we run all at once for api now, and one address, so then looks like we just need to remove local.* ones 👌

Reply:
@mordamax [Done.](https://github.com/paritytech/substrate-tip-bot/pull/50/commits/efcaad40ed3623ba527d3697a9b89991ba55260c)